arangod/VocBase/LogicalCollection.h

```diff
@@ -182,6 +182,7 @@ class LogicalCollection : public LogicalDataSource {
   bool isSmartEdgeCollection() const noexcept;
   bool isSatToSmartEdgeCollection() const noexcept;
   bool isSmartToSatEdgeCollection() const noexcept;
+  bool isSmartVertexCollection() const noexcept;
```

Comment:
There is no implementation for this new method in the Community Edition.
This may be ok if we don't call it, but it looks fishy. I suggest adding a stub implementation for the method as we did for the other methods above too.

Reply:
This method is only (and should only ever be) called from within EE code. I prefer to not have a definition in the community, so that we get a linker error in case this is ever called within a CE build.

Reply:
Ok. The other methods above do have an implementation though.
Not saying that these should be adjusted now. Just for information.